demos/covid-19/covid_19_get_r0_1/model/parts/get_r0_model.py

```diff
@@ -0,0 +1,71 @@
+import numpy as np 
+import pandas as pd
+
+
+def get_current_timestep(cur_substep, s):
```

Comment:
You should use meaningful names. I don't know if `s`means state, step, substep, etc.